yml/imsim.yaml

```diff
@@ -1205,14 +1205,14 @@ tables:
     description: Error in right ascension, measured on g-band.
     mysql:datatype: FLOAT
     fits:tunit: deg
-    ivoa:ucd: pos.eq.raErr
+    ivoa:ucd: stat.error;pos.eq.ra
   - name: g_ra_dec_Cov
     "@id": "#Object.g_ra_dec_Cov"
     datatype: float
     description: Covariance between right ascension and declination, measured on g-band.
     mysql:datatype: FLOAT
     fits:tunit: deg^2
-    ivoa:ucd: pos.eq.ra_dec_Cov
+    ivoa:ucd: stat.covariance;pos.eq.ra;pos.eq.dec
```

Comment:
Same comment; let's use "pos.eq" instead of naming them separately.

Reply:
Same as other comment. I believe we decided to list both positional measurements separately.